scripts/runProduction.sh

```diff
@@ -1,5 +1,8 @@
+#!/bin/bash
 echo "Running Production Site"
 
+set -eux
```

Comment:
Could you add a comment explaining this

Reply:
I think this is pretty common in shell scripts (and it's considered good practice): [https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425](https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425).

You can also see variations on this theme in `runProduction.sh`, `createIntegrationPgDb.sh` and `server/register-mui-styles/scripts/gen-mui-registration.sh` (although we can probably delete that last one now dark mode is merged(?)).